From 06b94cd6b65941a001b796c776a07236a0404ea7 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 14 Oct 2018 21:51:29 +0200
Subject: [PATCH 1/1] sandbox: README: use setenv ethrotate no

If we want to control which network interface is actually used, we have to
issue 'setenv ethrotate no'. If ethrotate is not set any interface may be
used.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 board/sandbox/README.sandbox | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/board/sandbox/README.sandbox b/board/sandbox/README.sandbox
index ee4421801b..0f59da4400 100644
--- a/board/sandbox/README.sandbox
+++ b/board/sandbox/README.sandbox
@@ -248,6 +248,7 @@ DHCP
 ....
 
 setenv autoload no
+setenv ethrotate no
 setenv ethact eth1
 dhcp
 
@@ -255,6 +256,7 @@ PING
 ....
 
 setenv autoload no
+setenv ethrotate no
 setenv ethact eth1
 dhcp
 ping $gatewayip
@@ -263,6 +265,7 @@ TFTP
 ....
 
 setenv autoload no
+setenv ethrotate no
 setenv ethact eth1
 dhcp
 setenv serverip WWW.XXX.YYY.ZZZ
@@ -287,6 +290,7 @@ operation being tested on the lo interface.
 TFTP
 ....
 
+setenv ethrotate no
 setenv ethact eth5
 tftpboot u-boot.bin
 
-- 
2.19.1

